src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
```

Comment:
Try modifying the code to return the adder function once.

Reply:
then it won't be a closure?

Reply:
> then it won't be a closure?

It remains the same, but it's unnecessary